flutter/lib/src/sentry_flutter.dart

```diff
@@ -167,8 +167,7 @@ mixin SentryFlutter {
       integrations.add(LoadImageListIntegration(channel));
     }
     final renderer = options.rendererWrapper.getRenderer();
-    if (renderer == FlutterRenderer.skia ||
-        renderer == FlutterRenderer.canvasKit) {
+    if (!platformChecker.isWeb || renderer == FlutterRenderer.canvasKit) {
```

Comment:
The HTML renderer is the only renderer where screenshots do not work. Therefore, it should be enough to check for `!web` and the CanvasKit renderer

Reply:
Do you know if we have it documented somewhere that HTML renderer isn't supported for screenshot? 

Might be good to improve the docs for that as well some time since the docs on sentry.io don't specify that

Reply:
@denrase might know that, since he wrote it